src/fontra/client/web-components/menu-panel.js

```diff
@@ -86,50 +91,97 @@ class MenuPanel extends SimpleElement {
       gap: 0.5em;
       justify-content: space-between;
     }
+
+    .submenu-icon {
+      width: 10px;
+      height: 14px;
+    }
   `;
 
-  constructor(menuItems, position, positionContainer) {
+  constructor(
```

Comment:
Can we simplify the constructor? I'm thinking all but the first arguments should go into an options object:

```js
  constructor(menuItems, options) {
```
Since many of these are optional this should simplify things. Also, it makes it easier to add more if we need.

Reply:
Yes, you're right.